src/datepicker/datepicker-month-view.ts

```diff
@@ -48,14 +48,14 @@ import {DayTemplateContext} from './datepicker-day-template-context';
       </div>
     </div>
     <ng-template ngFor let-week [ngForOf]="month.weeks">
-      <div *ngIf="!isCollapsed(week)" class="ngb-dp-week" role="row">
```

Comment:
Why did you remove the `role="row"` here ?

Reply:
Unfortunate rebase, thanks for the catch!